docs/examples/general/data_loading/dataloading_webdataset.ipynb

```diff
@@ -16,7 +16,7 @@
     "Data stored in WebDataset format can be read with `readers.webdataset` operator. The operator takes the following arguments:\n",
     "\n",
     "* `paths` the path (or list of paths) to the tar archives containing webdataset\n",
-    "* `index_paths` the path (or list of paths) to the respective index files containing data about the specifics of the tar files, created using `wds2idx` - a utility included with DALI. For the usage details please refer to `wds2idx -h`.\n",
+    "* `index_paths` the path (or list of paths) to the respective index files containing data about the specifics of the tar files, created using `wds2idx` - a utility included with DALI. For the usage details please refer to `wds2idx -h`. If not provided, it will be automatically inferred from the tar file, although it will take considerable time for big datasets.\n",
```

Comment:
```suggestion
    "* `index_paths` the path (or list of paths) to the respective index files containing data about the specifics of the tar files, created using `wds2idx` - a utility included with DALI. For the usage details please refer to `wds2idx -h`. If not provided, it will be automatically generated from the tar file, although it will take considerable time for big datasets.\n",
```

Reply:
same as above